orc8r/cloud/go/mproto/mproto.go

```diff
@@ -14,14 +14,31 @@
 package mproto
 
 import (
+	"crypto/md5"
+	"encoding/base64"
+
 	"magma/orc8r/cloud/go/mproto/protos"
 
 	"github.com/golang/protobuf/proto"
 )
 
-// MarshalManyDeterministic deterministically encodes a slice of protobuf messages,
+// HashManyDeterministic takes a collection of proto messages and returns a deterministic
+// base64-encoded md5 hash of the messages.
+func HashManyDeterministic(protosByID map[string]proto.Message) (string, error) {
+	digest, err := marshalManyDeterministic(protosByID)
+	if err != nil {
+		return "", err
+	}
+	// Convert to a constant-length hash of the encoded value.
+	sum := md5.Sum(digest)
+	digestHash := base64.StdEncoding.EncodeToString(sum[:])
+
+	return digestHash, nil
```

Comment:
```suggestion
	digest := base64.StdEncoding.EncodeToString(sum[:])

	return digest, nil
```

Reply:
Changed the initial variable name to `digestBytes` to avoid redeclaring the same variable as a different type @hcgatewood